build.gradle

```diff
@@ -13,7 +13,7 @@ ext {
     wordPressUtilsVersion = '3.0.0'
     wordPressLoginVersion = '1.0.0'
     aztecVersion = 'v1.6.2'
-    gutenbergMobileVersion = 'v1.84.0'
+    gutenbergMobileVersion = '5223-16130b31389066fbbb06afa116b5b3ab5591fa38'
```

Comment:
Friendly note: we'll need to update this to the newly created `1.84.1` tag before merging.